samples/v1.1/Elements/Image.BackgroundColor.json

```diff
@@ -2,35 +2,19 @@
 	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
 	"type": "AdaptiveCard",
 	"version": "1.0",
-	"body": [{
-		"type": "ColumnSet",
-		"columns": [{
-				"type": "Column",
-				"width": "stretch",
-				"items": [{
-					"type": "Image",
-					"backgroundColor": "#982374",
-					"url": "http://adaptivecards.io/content/cats/1.png"
-				}]
-			},
-			{
-				"type": "Column",
-				"width": "stretch",
-				"items": [{
-					"type": "Image",
-					"backgroundColor": "#128192",
-					"url": "http://adaptivecards.io/content/cats/1.png"
-				}]
-			},
-			{
-				"type": "Column",
-				"width": "stretch",
-				"items": [{
-					"type": "Image",
-					"backgroundColor": "#183774",
-					"url": "http://adaptivecards.io/content/cats/1.png"
-				}]
```

Comment:
Hey @dclaux, are the changes to teh test samples related to your changes at all? My only concern is a bad merge accidentally overwriting stuff in master for 1.2 ..

Reply:
I don't remember making any changes here.